bedrock/settings/base.py

```diff
@@ -105,12 +101,10 @@ def path(*args):
 PROD_DETAILS_JSON_REPO_PATH = config('PROD_DETAILS_JSON_REPO_PATH',
                                      default=path('product_details_json'))
 PROD_DETAILS_JSON_REPO_URI = config('PROD_DETAILS_JSON_REPO_URI',
-                                    default='https://github.com/mozilla/product-details-json.git')
+                                    default='https://github.com/mozilla-releng/product-details.git')
+PROD_DETAILS_JSON_REPO_BRANCH = config('PROD_DETAILS_JSON_REPO_BRANCH', default='production')
 # path to updated p-d data for testing before loading into DB
-PROD_DETAILS_TEST_DIR = str(Path(PROD_DETAILS_JSON_REPO_PATH).joinpath('product-details'))
-# if the repo is cloned it will be most up-to-date
-if Path(PROD_DETAILS_TEST_DIR).is_dir():
-    PROD_DETAILS_DIR = PROD_DETAILS_TEST_DIR
```

Comment:
There are still some references to `PROD_DETAIL_DIR` in `bedrock/firefox/tests/test_firefox_details.py` and `bedrock/firefox/tests/test_base.py`

Reply:
Yeah. It's a default setting in the `product_details` library. Using it in those tests is fine.

Reply:
I think with this removal it's no longer possible to use `product_details.storage.PDFileStorage` as `PROD_DETAILS_STORAGE`. The files stored in `PROD_DETAILS_TEST_DIR` could be updated by the management command, but they are never used.

Reply:
I think you're right. We used to need that so that things would work without the external DB, but since we moved to the SQLite distributed db we no longer need all of this. I should do some more cleanup to remove more of these settings and just hard-code `PDDatabaseStorage`.